src/interaction/index.ts

```diff
@@ -19,8 +19,22 @@ export function start(): void {
 }
 
 export function end(): void {
+    click.reset();
+    pointer.end();
     input.end();
     pointer.end();
     scroll.end();
     selection.end();
```

Comment:
Why are some thing not present here like unload.end()?

Reply:
Fixed it - made things consistent.